packages/mobile/src/components/details-tile/TrackMetadata.tsx

```diff
@@ -0,0 +1,55 @@
+import { useGetTrackById } from '@audius/common/api'
+import { useTrackMetadata } from '@audius/common/hooks'
+import type { ID } from '@audius/common/models'
+import { trpc } from '@audius/web/src/utils/trpcClientWeb'
+
+import { Flex, TextLink } from '@audius/harmony-native'
+
+import { MetadataRow } from './MetadataRow'
+
+const messages = {
+  album: 'Album'
+}
+
+type TrackMetadataProps = {
+  trackId?: ID
```

Comment:
Would argue that we shouldn't be rendering this component if we don't have a valid track ID, instead of asking the component to handle being passed an empty track id?